frontend/src/metabase/common/components/DataPicker/components/DataPickerModal.tsx

```diff
@@ -49,11 +48,26 @@ interface Props {
   onClose: () => void;
 }
 
-const QUESTION_PICKER_MODELS: CollectionItemModel[] = ["card"];
+const QUESTION_PICKER_MODELS: CollectionItemModel[] = [
+  "card",
+  "dataset",
+  "metric",
+];
 
-const MODEL_PICKER_MODELS: CollectionItemModel[] = ["dataset"];
-
-const METRIC_PICKER_MODELS: CollectionItemModel[] = ["metric"];
+const QUESITON_PICKER_MODEL_FILTER_OPTIONS = [
+  {
+    label: t`Metrics`,
+    value: "metric" as const,
+  },
+  {
+    label: t`Models`,
+    value: "dataset" as const,
+  },
+  {
+    label: t`Saved questions`,
+    value: "card" as const,
+  },
+];
```

Comment:
I think since `QUESITON_PICKER_MODEL_FILTER_OPTIONS` is defined outside a component, ttag might not work since the strings are defined before the locale is applied. An easy solution is to change this to a function, `getQuestionPickerModelFilterOptions`, that returns this array, and call this function instead of using the global variable